samples/drivers/opamp/output_measure/sample.yaml

```diff
@@ -17,6 +17,6 @@ tests:
     platform_allow:
       - frdm_mcxa156
       - frdm_mcxa346
-      - frdm_mcxa266
+      - frdm_mcxa366
```

Comment:
you need add `opamp` in frdm_mcxa366 support list and I find there is same problem with all frdm_mcxaxxx platforms